COPYING: update and install COPYING, NEWS and AUTHORS

rouault opened the pull request:
Fixes #2415

- Change mention of PROJ.4 to PROJ
- Change 'I' pronoun to Frank Warmerdam

kbevers commented on COPYING:
```diff
@@ -1,18 +1,37 @@
  --------------
 
- Copyright (c) 2000, Frank Warmerdam
+Copyright:             2008-2020, Even Rouault <even.rouault at spatialy.com>
```
```suggestion
Copyright:             2008-2020, Even Rouault <even.rouault at spatialys.com>
```

rouault replied:
good catch. can't even spell correctly my company name lol

danme74 commented:
Ok, fine. Any idea why the COPYING file isn't installed as part of the installation? (It usually is?)

kbevers commented on COPYING:
```diff
@@ -1,18 +1,37 @@
+Copyright:             2008-2020, Even Rouault <even.rouault at spatialys.com>
+                       2016-2020, Kristian Evers
+                       2016-2018, Thomas Knudsen
+                       2016-2018, SDFE
```
To be exact

```suggestion
                       2016-2018, Danish Agency for Data Supply and Efficiency
```

I guess the period should be extended to 2020 as well since the majority of my contributions is done on company time.

kbevers commented:
Thinking about it I am not sure about the list of copyright holders. Are we going to update the copyright info with every release? Is it good enough if we say that a list of contributors can be extracted from git if someone needs it? Looking for a low maintenance solution here. `COPYING` is unchanged for the last 20 years, I'd be happy to not touch it again for another 20 years :)

rouault commented:
> Thinking about it I am not sure about the list of copyright holders. Are we going to update the copyright info with every release? Is it good enough if we say that a list of contributors can be extracted from git if someone needs it?

OK, I removed the copyright holder list and replaced by "Copyright information can be found in source files."

kbevers reviewed (approved):
Looks good to me!
